hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestSimpleRequestController.java

```diff
@@ -115,6 +116,7 @@ public long heapSize() {
   }
 
   @Test
+  @SuppressWarnings("ArrayAsKeyOfSetOrMap")
```

Comment:
Since this is a test, let's just use TreeMap with Bytes.BYTES_COMPARATOR instead of HashMap to remove the warning?

Reply:
I thought I went back and undid my changes related to ArrayAsKeyOfSetOrMap while revising the patch but see that I left this one in place.